### ISSUE TITLE
Remove need for the properties file in the suite UI

### ISSUE THREAD

**Issue by LLambertRGi** [GitHubRGI/swagd#131]

**Comment by stevendlander** [comment]
Is this the project.properties file? If so, I think it is needed by Eclipse.

**Comment by LLambertRGi** [comment]
No, this is "geosuite.properties".  This is old code from Duff that had stuff like icon paths and tooltip strings.  This will be OBE when we ditch Swing for JavaFX.
